README.md

```diff
@@ -53,6 +53,38 @@ You can first localize the robot by initializing the pose and then setting a nav
 
 [![](media/localize.png)](media/localize.mp4)
 
+## Running the reconnaissance demos:
+
```

Comment:
Might want to add some text describing what this section is about, and what it demonstrates, and what each step does. This would be helpful for folks just reading this later offline or after/beyond the workshop session.

Reply:
Good point, thanks! See 1dc426384a8c20270d885c01a4da495e3763c213